pytest_pootle/fixtures/models/store.py

```diff
@@ -275,35 +273,6 @@ def _make_member_updates(store, member):
     _mark_unit_fuzzy(store.units[0], member)
 
 
-@pytest.fixture(scope='session')
```

Comment:
@phlax can we put this change to https://github.com/translate/pootle/pull/4585/commits/3738cca454a24f2163d7e38ae42a1a0ffaf5bb2b commit to show that it was moved?